src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -125,15 +159,27 @@ public SslClientAuthenticationOptions SslOptions
 
         public bool SupportsAutomaticDecompression => false;
         public bool SupportsProxy => false;
-        public bool SupportsRedirectConfiguration => false;
+        public bool SupportsRedirectConfiguration => true;
 
         private Dictionary<string, object?>? _properties;
         public IDictionary<string, object?> Properties => _properties ??= new Dictionary<string, object?>();
 
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && !_disposed)
+            {
+                _disposed = true;
+                _handlerStarted = false;
```

Comment:
We do not reset `_handler` back to `null` in `SocketsHttpHandler.Dispose` (equivalent of your `_handlerStarted`).
Is there a reason you're unsetting `_handlerStarted` here?

Reply:
Due to the dispose of handler:  https://github.com/kjpou1/runtime/blob/wasm-http-redirect/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs#L429

This is not an object so there is no dispose.  The unsetting too false was the next best thing.

Reply:
Removed from code in recent commits.